integration-test/README.md

```diff
@@ -0,0 +1,59 @@
+End to end storm integration tests
+==================================
+
+Running tests end-to-end
+------------------------
+Assumption:

```

Comment:
I would rename Assumption to Requirements, as it is a *requirement* to have this in order to run the tests